csharp/tools/MauiModelTester/NuGet.config

```diff
@@ -6,7 +6,6 @@
   <packageSources>
     <clear />
     <add key="NuGet Official" value="https://api.nuget.org/v3/index.json" />
-    <add key="ORT-Nightly" value="https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/nuget/v3/index.json" />
```

Comment:
Can you please sync with main so the checked in version has this commented out instead of removed?